src/Models/Constants/TelemetryConstants.ts

```diff
@@ -0,0 +1,191 @@
+                        countAssets: 'countAssets',
+                        hasCoordinates: 'hasCoordinates',
+                        hasDescription: 'hasDescription',
+                        pollingDelay: 'pollingDelay',
+                        sceneHash: CUSTOM_PROPERTY_NAMES.SceneHash
+                    }
+                },
+                ParseElement: {
+                    eventName: 'ParseConfig.ElementKpis',
+                    properties: {
+                        elementType: 'elementType',
+                        countMeshes: 'countMeshes',
+                        countAliases: 'countAliases',
+                        parentSceneHash: CUSTOM_PROPERTY_NAMES.ParentSceneHash
+                    }
+                },
+                ParseBehavior: {
+                    eventName: 'ParseConfig.BehaviorKpis',
+                    properties: {
+                        countAliases: 'countAliases',
+                        countElements: 'countElements',
+                        countDataSources: 'countDataSources',
+                        countWidgets: 'countWidgets',
+                        countWidgetGaugeType: 'countWidgetGaugeType',
+                        countWidgetLinkType: 'countWidgetLinkType',
+                        countWidgetPropertyType: 'countWidgetPropertyType',
+                        countVisuals: 'countVisuals',
+                        countVisualBadgeType: 'countVisualBadgeType',
+                        countVisualColorType: 'countVisualColorType',
+                        parentSceneHash: CUSTOM_PROPERTY_NAMES.ParentSceneHash
+                    }
+                }
+            }
+        }
+    },
+    /** builder section of the app */
+    Builder: {
+        /** scene list page */
+        SceneList: {
+            UserAction: {
+                /** creating a new scene */
+                CreateScene: {
+                    Cancel: {
+                        eventName: 'SceneList.CreateScene.Cancel'
+                    },
+                    Confirm: {
+                        eventName: 'SceneList.CreateScene.Confirm',
+                        properties: {
+                            hasCoordinates: 'hasCoordinates',
+                            hasDescription: 'hasDescription'
+                        }
+                    },
+                    Initiate: {
+                        eventName: 'SceneList.CreateScene.Initiate'
+                    }
+                },
+                /** delete an existing scene */
+                DeleteScene: {
+                    Cancel: {
+                        eventName: 'SceneList.DeleteScene.Cancel'
+                    },
+                    Confirm: {
+                        eventName: 'SceneList.DeleteScene.Confirm'
+                    },
+                    Initiate: {
+                        eventName: 'SceneList.DeleteScene.Initiate',
+                        properties: {
+                            itemIndex: 'itemIndex'
```

Comment:
why itemIndex is same for other type of actions below?

Reply:
because I want to define the properties that each event has, and I wanted the names of those properties to be consistent. They don't HAVE to match, but it's easier when you're querying to know. And it's flexible if we want to change them for some reason. I'm only pulling common names out to constants if they really need to match exactly everywhere for some reason (like the sceneHash)